tests/dialects/test_duckdb.py

```diff
@@ -536,7 +536,7 @@ def test_time(self):
             write={
                 "bigquery": "UNIX_TO_TIME(x / 1000)",
                 "duckdb": "TO_TIMESTAMP(x / 1000)",
-                "presto": "FROM_UNIXTIME(x / 1000)",
+                "presto": "FROM_UNIXTIME(CAST(x AS DOUBLE) / 1000)",
```

Comment:
interesting, was this a bug

Reply:
No, because `FROM_UNIXTIME` seems to cast to integer.